test/functional/test_framework/test_node.py

```diff
@@ -136,3 +136,7 @@ def node_encrypt_wallet(self, passphrase):
             time.sleep(0.1)
         self.rpc = None
         self.rpc_connected = False
+
+    def cli(self, args):
+        """Start the cli."""
+        return subprocess.Popen(['bitcoin-cli'] + args, stderr=self.stderr, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
```

Comment:
You might be able to hack something up based on `TestNodeCLI` from current `master` branch.